src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    key instead of full tuples in DML events (`space_insert`, `space_replace`,
+    `space_delete`). Otherwise, full tuples are logged. The option may be
+    useful in case tuples are big.
+]])
+
+M['audit_log.file'] = format_text([[
+    Specify a file for the audit log destination. You can set the `file` type
+    using the audit_log.to option. If you write logs to a file, Tarantool
+    reopens the audit log at SIGHUP.
+]])
+
+M['audit_log.filter'] = format_text([[
+    Enable logging for a specified subset of audit events. This option accepts
+    the following values:
+
+    - Event names (for example, password_change). For details, see Audit log
+    events.
+    - Event groups (for example, audit). For details, see Event groups.
+
+    The option contains either one value from Possible values section or a
+    combination of them.
+
+    To enable custom audit log events, specify the custom value in this option.
+]])
+
+M['audit_log.filter.*'] = format_text([[
+    Specify a subset of audit events to log by providing a value
+    from the allowed list of events or groups.
+]])
+
+M['audit_log.format'] = 'Specify a format that is used for the audit log.'
+
+M['audit_log.nonblock'] = format_text([[
+    Specify the logging behavior if the system is not ready to write.
+    If set to `true`, Tarantool does not block during logging if the system
+    is non-writable and writes a message instead. Using this value may
+    improve logging performance at the cost of losing some log messages.
+]])
+
+M['audit_log.pipe'] = format_text([[
+    Specify a pipe for the audit log destination. You can set the `pipe` type
+    using the audit_log.to option. If log is a program, its pid is stored in
+    the `audit.pid` field. You need to send it a signal to rotate logs.
+]])
+
+M['audit_log.spaces'] = format_text([[
+    The array of space names for which data operation events (`space_select`,
+    `space_insert`, `space_replace`, `space_delete`) should be logged. The array
+    accepts string values. If set to box.NULL, the data operation events are
+    logged for all spaces.
+]])
+
+M['audit_log.spaces.*'] = format_text([[
+    A specific space name in the array for which data operation events are
+    logged. Each entry must be a string representing the name of the space
+    to monitor.
+
+    Example:
+
+    `spaces: [bands, singers]`, only the events of `bands` and `singers` spaces
+    are logged.
+]])
+
+M['audit_log.to'] = 'Enable audit logging and define the log location.'
+
+M['audit_log.syslog'] = format_text([[
+    This module allows configuring the system logger (syslog) for audit
+    logs in Tarantool. It provides options for specifying the syslog server,
+    facility, and identity for logging messages.
+]])
+
+M['audit_log.syslog.facility'] = format_text([[
+    Specify a system logger keyword that tells syslogd where to send the
+    message. You can enable logging to a system logger using the
+    `audit_log.to` option.
+]])
```

Comment:
AFAIR, facility is not a kind of destination. It is more about a type of the application that issues log entries.

https://datatracker.ietf.org/doc/html/rfc3164#section-4.1.1

Reply:
Maybe like this?
```lua
I['audit_log.syslog.facility'] = format_text([[
    Define the syslog facility, which indicates the type of application
    generating the log entries (e.g., kernel, user-level, or system daemon).
    To enable syslog logging, set `audit_log.to` to `syslog`.
]])
```